integrated_channels/integrated_channel/exporters/learner_data.py

```diff
@@ -496,6 +430,7 @@ def get_learner_assessment_data_records(
     ):
         """
         Generate a learner assessment data transmission audit with fields properly filled in.
+        Returns a list of LearnerDataTransmissionAudit objects.
```

Comment:
Do you think it's worth adding params here as you did above a few times?

Reply:
Yes, will do.